File: Fundamentals with JS/T10 - Functions/exercise/04. Odd and Even Sum.js
function oddAndEvenSum(num) {
    let evenSum = 0;
    let oddSum = 0;
    let numToString = num.toString()

    for (let i = 0; i < numToString.length; i++) {
        if (numToString[i] % 2 === 0) {
            evenSum += Number(numToString[i]);
        } else {
            oddSum += Number(numToString[i]);
        }
    }
    console.log(`Odd sum = ${oddSum}, Even sum = ${evenSum}`);
}
oddAndEvenSum(1000435);
oddAndEvenSum(3495892137259234);

File: Fundamentals with Python/T09 - Functions/exercise/04. Odd and Even Sum.py
number = input()

def odd_even_sum(number):
    sum_odd = 0
    sum_even = 0

    for digit in number:
        digit = int(digit)
        if digit % 2 == 0:
             sum_even += digit
        else:
            sum_odd += digit

    return f'Odd sum = {sum_odd}, Even sum = {sum_even}'

print(odd_even_sum(number))